0.data_analysis/nbconverted/plates_1_2_sig_test_nf1_sc_all_cellprofiler.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Obnibus and post hoc testing with Anova and scheffe's test
```

Comment:
Just a recommendation, it might be nice to have either a quick sentence or links (or both) to describe what Obnibus and Post Hoc are so that in case someone isn't familiar can be directed to a place to learn more.

Reply:
Added links